Beyond08-SOW.md

```diff
@@ -0,0 +1,473 @@
+# Red Hat Beyond 08
+
+Statement of Work (SOW)
+***
+
+## Pre-requirements
+
+You will be working on a project during this course.
+
+The following tech stack will be used.
+
+If you don't have knowledge/experience in any of them, it is worthwhile to look
+them up. The more you know the easier it will be.
+
+* Linux OS - able to use basic command line tools such as cd, ls, cat, grep,
+  less, vi, ...
+* Python - if/else, for loops, functions, classes
+* Django - create a basic or sample project
+* Git and GitHub - create a repo, commit, push, pull requests (PR), merge
+* HTML/CSS (Bootstrap) - design a basic web page from scratch, including using
+  CSS classes for basic formatting
+* Testing- unit tests and functional tests with pytest
+* Virtualization - Spin up a VM using Vagrant and VirtualBox
+* Databases - Basic familiarity with database concepts
+
+## Milestones
+
+### Demo 1 (November 13th): Project selection and setting up a DEV environment
+
+The main purpose of the demo is to motivate you to come together as a project
+development team and establish your shared working environment and process. As
+such, we will monitor when and how you do the work of preparing for the demo,
+and the process is as important (If not more so) as the final results you will
```

Comment:
s/", and"/"; note that"

Reply:
@Omeramsc ^^

Reply:
LGTM, but I'll keep "as important" instead of "equal or more important".

Reply:
Ack